assets/README.md

```diff
@@ -0,0 +1,109 @@
+
+
+## Table of Contents
+
+- [Insert project name](#insert-project-name)
+  - [Project overview](#markdown-header-project-overview)
+  - [Table of Contents](#markdown-header-table-of-contents)
+  - [Local environment](#markdown-header-local-environment)
+    - [Docker - Docker4Drupal](#markdown-header-docker-docker4drupal)
+    - [Config splits:](#markdown-header-config-splits)
+  - [Theming and frontend](#markdown-header-theming-and-frontend)
+    - [Theme workflow and compiling](#markdown-header-theme-workflow-and-compiling)
+    - [Theme best practices](#markdown-header-theme-best-practices)
+  - [Integrations](#markdown-header-integrations)
+  - [Hosting and deployment](#markdown-header-hosting-and-deployment)
+    - [Deployment procedure](#markdown-header-deployment-procedure)
+  - [Misc notes](#markdown-header-misc-notes)
+  - [Credits](#markdown-header-credits)
+  - [License](#markdown-header-license)
+
+
+## Local environment
+
+Write here what is the **recommended** setup process to get the site up and running. Note that the site configuration _should not allow_ to accidentally push data to production systems if the developer does not follow the instructions here, so take care that all the informations reported here must be considered _optional_.
+
+
+#### Config splits
+
+Define the config environment used to create config splits. An example could be:
+
+- **Environment: Dev** _(Machine name: `environment_dev`)_: Enabled by default (disabled in `settings.ramsalt.prod.php`), it disables most cache modules,
+  enables Devel module and database logging.
+
+
+## Theming and frontend
+| FRONTEND INFO   |             |
+| --------------- | ----------- |
+| **Base theme**      | BaseTheme Name |
+| **Module bundler**  | Webpack 5 |
+| **Node version**    | v14.15.4 |
+| **CSS framework**   | Bootstrap 4.6 |
+| **JS framework**    | Vanilla JS - jQuery 3.5.1 |
+| **Additional notes**| None |
+
+#### Theme workflow and compiling
+
+Explain the command and the process of the theme. An example could be:
+
+**Compilation**: Run gulp from the awesome_theme folder.
+
+
+#### Theme best practices
+
+A description of linting, component based design and how to behave in the theme
+
+
+## Integrations
+
+Define any 3rd-party integration and any notes on it here
+
+
+## Hosting and deployment
+
+This project is hosted via Wodby and connected to CircleCI, for managing the `post-script` commands, we use the `wodby.yml` file in the docroot.
+Note that the production environment should always be set to the `master` branch.
+
+
+#### Deployment procedure
+
+The best practice to consider when deploying is to do your changes on `develop` branch, make a pull-request and when approved by the PM, make a database backup of the `prod` instance in Wodby, and merge with `master`.
+
+
+## Misc notes
+
+Any miscellaneous non-categorable info you may need to add
+
+
+- - -
+## Credits
```

Comment:
Does this section make sense at all?

1. We have it already on `composer.json`
2. 90% of the cases it will be the same name 4 times. 😅

Reply:
yeah let's remove it then